scripts/collect_model_urls.py

```diff
@@ -0,0 +1,22 @@
+import pathlib
+import re
+import sys
+
+MODEL_URL_PATTERN = re.compile(r"https://download.pytorch.org/models/.*?[.]pth")
+
+
+def main(root):
+    model_urls = set()
```

Comment:
Nice. This needs to be a set because of the `prototype` area. Note that we shouldn't exclude the prototype area from the scan because in there, there might be prototype models that still need to be tested. There is an example of that in a PR for the ViT model. So what you do here is great.

Just add a comment that this needs to remain as is to handle duplicate URLs.